addons/sys_core/CfgVehicles.hpp

```diff
@@ -0,0 +1,25 @@
+class CfgNonAIVehicles {
+    class GVAR(connectorWireSegment) {
+        scope = 2;
+        displayName = "Connector Wire";
```

Comment:
I assume this and below Helper names are not visible anywhere, thus don't have to be translated?

Reply:
afaik they should only be visible to a Zeus in the object list if the helper is selected as editable. So no.